commit/proximals.pyx

```diff
@@ -9,10 +8,27 @@ import sys
 @cython.wraparound(False)
 @cython.profile(False)
 
-## Regularisers for NNLSL1
-# Proximal
-cpdef prox_nnl1(np.ndarray[np.float64_t] x, double lam, int compartment_start, int compartment_size) :
+## Regularisers for NNLSLp
+#####  #####   ####  #    # # #    #   ##   #       ####
```

Comment:
Do we need these comments?

Reply:
Not at all. I forgot to clean this file.